### ISSUE TITLE
RCON port cannot be exposed through docker

### ISSUE THREAD

**Issue by egladil** [OpenFactorioServerManager/factorio-server-manager#240]
In order to expose a port through docker it needs to be known before starting the container. But since the RCON port is always randomized this is not possible. It would be useful to have a setting of some kind for forcing the RCON port to a specific port number

**Comment by mroote** [comment]
Good idea we should have an optional flag to specify the RCON port.

**Comment by daggers953** [comment]
Yes i need this to use the factorio chat bot. I need to set the rcon port.
https://github.com/Corbris/FactorioChatBot

**Comment by Mirdinus** [comment]
Hi @mroote,
I think that issue should be changed to bug as rcon_port parameter is ignored and randomized every time config is loaded
FactorioRconPort is assigned random value (config.go#L176) every time loadServerConfig is called


https://github.com/OpenFactorioServerManager/factorio-server-manager/blob/1d42f87e55aa246001e9b44e0caa5ec8456407ce/src/bootstrap/config.go#L176
